qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -673,13 +673,21 @@ def _decomposer_2q_from_target(self, target, qubits, approximation_degree):
         # available instructions on this qubit pair, and their associated property.
         available_2q_basis = {}
         available_2q_props = {}
+
+        def replace_parameterized_gate(op):
+            if isinstance(op, RXXGate) and isinstance(op.params[0], Parameter):
```

Comment:
Do you think it makes sense to check if `op.param[0] == pi / 2` too? I'm not sure it matters because the other selection code will work with that case though because it can reason about the unitary for the gate with a fixed angle.